psuutil/main.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python
+#
+# main.py
+#
+# Command-line utility for interacting with PSU in SONiC
+#
+
+try:
+    import sys
+    import os
+    import subprocess
+    import click
+    import imp
+    import syslog
+    import types
+    import traceback
+    from tabulate import tabulate
+except ImportError as e:
+    raise ImportError("%s - required module not found" % str(e))
+
+VERSION = '1.0'
+
+SYSLOG_IDENTIFIER = "psuutil"
+PLATFORM_SPECIFIC_MODULE_NAME = "psuutil"
+PLATFORM_SPECIFIC_CLASS_NAME = "PSUutil"
```

Comment:
Suggest changing case of class name to `PsuUtil` to match CamelCase of `SfpUtil`